sdk/monitor/azquery/metrics_client_test.go

```diff
@@ -42,10 +42,11 @@ func TestQueryResource_BasicQuerySuccess(t *testing.T) {
 	client := startMetricsTest(t)
 	timespan := azquery.TimeInterval("PT12H")
 	res, err := client.QueryResource(context.Background(), resourceURI,
-		&azquery.MetricsClientQueryResourceOptions{Timespan: to.Ptr(timespan),
+		&azquery.MetricsClientQueryResourceOptions{
+			Timespan:        to.Ptr(timespan),
 			Interval:        to.Ptr("PT1M"),
 			MetricNames:     nil,
-			Aggregation:     to.Ptr("Average,count"),
+			Aggregation:     []*azquery.AggregationType{to.Ptr(azquery.AggregationTypeAverage), to.Ptr(azquery.AggregationTypeCount)},
```

Comment:
`to.SliceOfPtrs()`

Reply:
fixed in [ffaf2f4](https://github.com/Azure/azure-sdk-for-go/pull/19981/commits/ffaf2f45111bb13afbe765ed56b34bd1fe047e96)